seedlet/assets/sass/base/_editor.scss

```
@@ -19,7 +19,7 @@ body {
 // Links styles
 a {
 	border-bottom: 1px solid var(--global--color-secondary);
-	color: var(--global--color-primary);
+	color: var(--wp--style--color--link);
```

As an alternative to defining it in our abstracts, something like this should work?

```suggestion
	color: var( --wp--style--color--link, var(--global--color-primary) );
```

Not sure how much cleaner this is, but maybe it feels more contained and consistent?

---

Yeah, that's probably cleaner. I'll give it a shot.